  controller_list:
    - name: "wsg50"
      action_ns: gripper_control
      type: WeissGripperCmd
      joints:
        - wsg_50_base_joint_gripper_left
        - wsg_50_base_joint_gripper_right

